Sources/ClientRuntime/Networking/Http/HttpContent.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+
+/// A wrapper around a StreamReader that conforms to the IStreamable protocol
+/// This is used to pass the StreamReader to the CRT HttpMessage
+public class HttpContent: IStreamable {
```

Comment:
Can we add docs for all public methods in this class?

Reply:
I thought without docs, protocol methods automatically gets protocol docs but no :(

Reply:
done